### PR TITLE
[T6A3][T13-C2] Aloisius Stephen

### PR DESCRIPTION
**Pull request by aloisiusStephen**

### CODE REVIEW

File: src/seedu/addressbook/storage/StorageStub.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.storage.StorageFile.InvalidStorageFilePathException;
+import seedu.addressbook.storage.StorageFile.StorageOperationException;
+import seedu.addressbook.storage.jaxb.AdaptedAddressBook;
+
```
Comment (KulaniT) [inline src/seedu/addressbook/storage/StorageStub.java]: Please add header comments

File: src/seedu/addressbook/storage/StorageStub.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+public class StorageStub extends Storage {
+
+    public final String path;
```
Comment (KulaniT) [inline src/seedu/addressbook/storage/StorageStub.java]: why public?

**Comment by KulaniT** [comment]
@aloisiusStephen  Some comments added. Please ack & close the PR after reading comments.

**Comment by aloisiusStephen** [comment]
thanks for the comments!

**Pull request closed without merging**